### PR TITLE
enable material m3 into google_maps example app

### PR DESCRIPTION
**Pull request by AdamMusa**
Enable material m3 in this app  and added minSdkVersion 20 to fix a small bug.

before enabling material 3 android and iOS screenshots

before_ios

before

After enabling material 3 android and iOS screenshots

after_ios

after

## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Contributors Guide]: https://github.com/flutter/samples/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: google_maps/lib/main.dart [hunk 1]
```diff
@@ -38,10 +38,16 @@ class _MyAppState extends State<MyApp> {
...
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+        useMaterial3: true,
+      ),
       home: Scaffold(
         appBar: AppBar(
           title: const Text('Sydney'),
           backgroundColor: Colors.green[700],
+
```
Comment (domesticmouse) [inline google_maps/lib/main.dart:50]: I'm curious what this white space is for. I suspect it'll fail format check.

Comment (AdamMusa) [reply google_maps/lib/main.dart:50]: > 

Oh it's just by mistake. Let's me fix

**Comment by domesticmouse** [comment]
Can you also add before and after screenshots to the PR?

**Comment by domesticmouse** [comment]
Oh and run `dart format`

**Comment by AdamMusa** [comment]
> Can you also add before and after screenshots to the PR?

Yeah of course

**Comment by AdamMusa** [comment]
> Oh and run `dart format`

Ok

**Review by domesticmouse — APPROVED** [review]
Thank you! 🙏

**Pull request merged**